packages/main/src/RadioButton.ts

```diff
@@ -0,0 +1,536 @@
+	 * @type {sap.ui.webc.base.types.ValueState}
+	 * @defaultvalue "None"
+	 * @name sap.ui.webc.main.RadioButton.prototype.valueState
+	 * @public
+	 */
+	@property({ type: ValueState, defaultValue: ValueState.None })
+	valueState!: ValueState;
+
+	/**
+	 * Defines the name of the component.
+	 * Radio buttons with the same <code>name</code> will form a radio button group.
+	 *
+	 * <br><br>
+	 * <b>Note:</b>
+	 * The selection can be changed with <code>ARROW_UP/DOWN</code> and <code>ARROW_LEFT/RIGHT</code> keys between radio buttons in same group.
+	 *
+	 * <br><br>
+	 * <b>Note:</b>
+	 * Only one radio button can be selected per group.
+	 *
+	 * <br><br>
+	 * <b>Important:</b> For the <code>name</code> property to have effect when submitting forms, you must add the following import to your project:
+	 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+	 *
+	 * <br><br>
+	 * <b>Note:</b> When set, a native <code>input</code> HTML element
+	 * will be created inside the component so that it can be submitted as
+	 * part of an HTML form.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.name
+	 * @public
+	 */
+	@property()
+	name!: string;
+
+	/**
+	 * Defines the form value of the component.
+	 * When a form with a radio button group is submitted, the group's value
+	 * will be the value of the currently selected radio button.
+	 * <br>
+	 * <b>Important:</b> For the <code>value</code> property to have effect, you must add the following import to your project:
+	 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.value
+	 * @public
+	 */
+	@property()
+	value!: string;
+
+	/**
+	 * Defines whether the component text wraps when there is not enough space.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>None</code> - The text will be truncated with an ellipsis.</li>
+	 * <li><code>Normal</code> - The text will wrap. The words will not be broken based on hyphenation.</li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.main.types.WrappingType}
+	 * @defaultvalue "None"
+	 * @name sap.ui.webc.main.RadioButton.prototype.wrappingType
+	 * @public
+	 */
+	@property({ type: WrappingType, defaultValue: WrappingType.None })
+	wrappingType!: WrappingType;
+
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.accessibleName
+	 * @public
+	 * @since 1.6.0
+	 */
+	@property()
+	accessibleName!: string;
+
+	/**
+	 * Defines the IDs of the elements that label the component.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.accessibleNameRef
+	 * @public
+	 * @since 1.1.0
+	 */
+	@property()
+	accessibleNameRef!: string;
+
+	@property({ type: String, defaultValue: "-1", noAttribute: true })
```

Comment:
minor: adding type is required for slots and for properties with type different than String, but for properties from type String it's optional